Resources/doc/reference/extensions.rst

```diff
@@ -52,6 +52,11 @@ Set the *global* attribute to *true* and the extension will be added to all admi
                 tags:
                     - { name: sonata.admin.extension, global: true }
 
+            app.important.extension:
+                class: AppBundle\Admin\Extension\ImportantAdminExtension
+                tags:
+                    - { name: sonata.admin.extension, priority: 5 }
+
```

Comment:
They seem to be using krsort.

ref: [](http://symfony.com/blog/new-in-symfony-3-2-compiler-passes-improveme)nts